src/sentry/processing/realtime_metrics/redis.py

```diff
@@ -80,3 +84,59 @@ def increment_project_duration_counter(
             pipeline.hincrby(key, duration, 1)
             pipeline.pexpire(key, self._histogram_ttl)
             pipeline.execute()
+
+    def get_lpq_projects(self) -> Set[int]:
+        """
+        Fetches the list of projects that are currently using the low priority queue.
+
+        Returns a list of project IDs.
+        """
+        return set(
+            filter(
+                None,
+                {
+                    _to_int(project_id_raw)
+                    for project_id_raw in self.cluster.smembers(LPQ_MEMBERS_KEY)
+                },
+            )
```

Comment:
I agree with Floris that this function shouldn't filter anything; project ids that aren't (string representation of) ints are indicative of a bug somewhere and shouldn't be silently skipped.

Reply:
the drawback of me breaking this PR up into two is that yep, @flub brought this up a few times in https://github.com/getsentry/sentry/pull/28714#discussion_r713849046.

i made changes there to remove calls to `_to_int` in that PR, but it meant that this still remained here. i'll backport some of those changes into this PR.

Reply:
ported these changes over and made a comment in the parent PR that might be of interest: https://github.com/getsentry/sentry/pull/28714#discussion_r718850558

tl;dr we can let the automated bits of this system fail instead of trying too hard to keep it running since we have the manual kill switches as a fallback if that happens

@loewenheim could i bother you to take a look at the ported changes?